index.html

```diff
@@ -37,6 +37,48 @@
     </nav>
 
 
+    <main>
+        <div class="theme">
```

Comment:
Let's find a more suitable name for this. `theme` doesn't really make a lot of sense, since here we have a login/authentication card.

I would suggest you rename this to `card`.